lib/livebook_web/live/output.ex

```diff
@@ -142,15 +142,19 @@ defmodule LivebookWeb.Output do
     assigns = %{message: message}
 
     ~H"""
-    <div class="whitespace-pre-wrap font-editor text-gray-500"><%= ansi_string_to_html(@message) %></div>
+    <div class="whitespace-pre-wrap font-editor text-gray-500" role="complementary" aria-label="error">
+      <%= ansi_string_to_html(@message) %>
+    </div>
```

Comment:
Nit: this has `whitespace-pre-wrap`, so we don't want to introduce additional space, hence it should be all in a single line :) (same below)